src/main/java/se/sundsvall/emailreader/api/EmailResource.java

```diff
@@ -52,30 +50,22 @@ class EmailResource {
 		this.service = service;
 	}
 
-	@Operation(
-		summary = "Get a list of emails",
-		responses = {
-			@ApiResponse(
-				responseCode = "200",
-				description = "Ok",
-				useReturnTypeSchema = true)
-		})
-	@GetMapping()
-	public ResponseEntity<List<Email>> getAllEmails(@PathParam("municipalityId") final String municipalityId, @PathParam("namespace") final String namespace) {
+	@Operation(description = "Get a list of emails")
+	@ApiResponse(responseCode = "200", description = "OK", useReturnTypeSchema = true)
+	@GetMapping("/{namespace}")
+	ResponseEntity<List<Email>> getAllEmails(
+		@PathVariable("municipalityId") @ValidMunicipalityId final String municipalityId,
+		@PathVariable("namespace") final String namespace) {
 		return ResponseEntity.ok(service.getAllEmails(municipalityId, namespace));
 	}
 
-	@Operation(
-		summary = "Delete an email by messageID",
-		responses = {
-			@ApiResponse(
-				responseCode = "204",
-				description = "No content")
-		})
-	@DeleteMapping(path = "{id}")
-	public ResponseEntity<Void> deleteEmail(
-		@Parameter(name = "id", description = "Email message ID", example = "81471222-5798-11e9-ae24-57fa13b361e1") @ValidUuid @PathVariable("id") final String id) {
-		service.deleteEmail(id);
+	@Operation(description = "Delete an email by id")
+	@ApiResponse(responseCode = "204", description = "No content")
+	@DeleteMapping(path = "/{id}")
+	ResponseEntity<Void> deleteEmail(
+		@PathVariable("municipalityId") @ValidMunicipalityId final String municipalityId,
```

Comment:
I think it's nice when we add descriptions and examples to API-parameters. 

Something like this maybe?:
`@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId`